### PR TITLE
review changes

### PR DESCRIPTION
**Pull request by p0lyn0mial**
**What this PR does / why we need it**:  n/a

**Release note**:

```
NONE
```

### CODE REVIEW

File: api/cmd/rbac-generator/main.go [hunk 1]
```diff
@@ -170,39 +167,31 @@ func main() {
...
 			}
 		})
 	}
 
 	// This group is running the actual controller logic
 	{
 		g.Add(func() error {
-			go ctrl.Run(ctrlCtx.runOptions.workerCount, ctrlCtx.stopCh)
-			<-ctrlCtx.stopCh
+			// controller will return iff ctrlCtx is stopped
```
Comment (zreigz) [inline api/cmd/rbac-generator/main.go:186]: iff -> if

File: api/cmd/rbac-generator/main.go [hunk 2]
```diff
@@ -142,22 +139,22 @@ func main() {
 	for _, seedClusterProvider := range ctrlCtx.allClusterProviders {
 		seedClusterProvider.StartInformers(ctrlCtx.stopCh)
 		if err := seedClusterProvider.WaitForCachesToSync(ctrlCtx.stopCh); err != nil {
-			glog.Fatalf("failed to sync cache: %v", err)
+			glog.Fatalf("closing the controller, failed to sync cache: %v", err)
```
Comment (zreigz) [inline api/cmd/rbac-generator/main.go]: It seems like capital letters are used for logs: closing  -> Closing

File: api/cmd/rbac-generator/main.go [hunk 3]
```diff
@@ -170,39 +167,31 @@ func main() {
...
-			glog.Errorf("Stopping internal http server: %v", err)
-			timeoutCtx, cancel := context.WithTimeout(ctx, time.Second)
+			glog.Infof("Stopping internal HTTP metrics server, err = %v", err)
+			timeoutCtx, cancel := context.WithTimeout(context.Background(), time.Second)
 			defer cancel()
 
-			glog.Info("Shutting down the internal http server")
 			if err := s.Shutdown(timeoutCtx); err != nil {
-				glog.Error("failed to shutdown the internal http server gracefully:", err)
+				glog.Errorf("failed to shutdown the internal HTTP server gracefully, err = %v", err)
```
Comment (zreigz) [inline api/cmd/rbac-generator/main.go]: failed -> Failed

**Review by zreigz — CHANGES_REQUESTED** [review]
small remarks, rest LGTM :)
thanks

File: api/cmd/rbac-generator/main.go (hunk 1 above)
Comment (p0lyn0mial) [reply api/cmd/rbac-generator/main.go:186]: iff as "if and only if" :) https://en.wikipedia.org/wiki/If_and_only_if#Origin_of_iff_and_pronunciation

**Comment by p0lyn0mial** [comment]
@zreigz done.

File: api/cmd/rbac-generator/main.go (hunk 1 above)
Comment (zreigz) [reply api/cmd/rbac-generator/main.go:186]: good to know, thanks

**Pull request merged**